src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -59,6 +64,38 @@ public void initializeAuthedSpeechlet() throws URISyntaxException {
     }
 
     @Test
+    public void getStopDetails() throws SpeechletException, URISyntaxException {
```

Comment:
@philipmw I was trying to write a unit test for this, but I can't seem to get it to pass - it currently fails with `com.amazon.speech.speechlet.SpeechletException: Error getting stop details for 6497`.

What am I missing for the proper setup here?